### PR TITLE
Blazor Custom Elements support is no longer experimental

### PR DESCRIPTION
**Pull request by mkArtakMSFT**

### CODE REVIEW

File: src/BlazorCustomElements/README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# The below content is now obsolete. Please refer to Blazor documentation if you'd like to use Custom Elements
+## The previously experimental Microsoft.AspNetCore.Components.CustomElements package for building standards based custom elements with Blazor is no longer experimental and is now part of [.NET 7 release](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-7-preview-6/#blazor-custom-elements-no-longer-experimental). You can learn more about how to use Custom Elements in Blazor from our [documentation](https://learn.microsoft.com/en-us/aspnet/core/blazor/components/?view=aspnetcore-7.0#blazor-custom-elements)
```
Comment (danroth27) [inline src/BlazorCustomElements/README.md:2]:
```suggestion
## The previously experimental Microsoft.AspNetCore.Components.CustomElements package for building standards based custom elements with Blazor is no longer experimental and is now part of [.NET 7](https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-7-preview-6/#blazor-custom-elements-no-longer-experimental). You can learn more about how to use Custom Elements in Blazor from our [documentation](https://learn.microsoft.com/aspnet/core/blazor/components#blazor-custom-elements)
```

**Review by danroth27 — APPROVED** [review]
One minor comment.

**Comment by bxjg1987** [comment]
How to return parameters from the blazer component to an external system?
How can events triggered by blazer components be received externally?

**Comment by MackinnonBuck** [comment]
@bxjg1987,

If you have a Blazor component registered as a custom element, and the component has an `EventCallback` or `EventCallback<T>` parameter, you can pass a JavaScript function to fulfill that parameter's value by using the property syntax (see https://learn.microsoft.com/aspnet/core/blazor/components/?view=aspnetcore-7.0#pass-parameters). This allows the Blazor component to invoke the `EventCallback` and communicate back to the JavaScript side.

**Pull request merged**